docs/source/user-guide/guide_data_dictionaries.rst

```diff
@@ -50,7 +50,7 @@ The "name" should match the datastore column name. These are derived from the co
 
 title
 ^^^^^
-This is usually the column header from the data file, but if the data file uses abbreviated column headings, this is where you can supply a more human readable and clear display title. This value will also be used for column headings when users export a filtered subset of results as a csv file.
+This is usually the column header from the data file, but if the data file uses abbreviated column headings, this is where you can supply a more human readable and clear display title. Depending on your :ref:`configuration<_guide_data_dictionary_config>`, this value may also be used for column headings when users export a filtered subset of results as a csv file.
```

Comment:
I am not seeing the link appear in https://dkan--4264.org.readthedocs.build/en/4264/user-guide/guide_data_dictionaries.html#title (might be a rtd bug?)

Reply:
Fixed.